packages/editor/src/toolbar/popups/color-picker.tsx

```diff
@@ -171,13 +171,16 @@ export function ColorPicker(props: ColorPickerProps) {
         ) : null}
         <Flex>
           <Flex
-            className="hide-scrollbar"
```

Comment:
We hide the scrollbar on mobile devices where the colors are horizontally scroll-able.